src/web/wms/src/main/resources/GeoServerApplication.properties

```diff
@@ -46,7 +46,7 @@ KMLLayerConfigPanel.kmlFormatSettings           = KML Format Settings
 
 StyleEditPage.description = Edit the current style. The editor can provide syntax highlighting and automatic formatting. Click on the "validate" button to verify the style is a valid SLD document.
 StyleEditPage.name        = Name
-StyleEditPage.title       = Style Editor
+StyleEditPage.title       = Style Editor - {0}
```

Comment:
This seems to be causing an issue with the page/tab title - the wildcard replacement does not seem to be working there:
![screen shot 2016-08-15 at 9 03 10 am](https://cloud.githubusercontent.com/assets/9271497/17670589/94114b5c-62c7-11e6-9039-2274e56db6b1.png)

(Works fine in the UI though)